packages/protocol/contracts/libs/LibConstants.sol

```diff
@@ -31,4 +31,6 @@ library LibConstants {
 
     bytes32 public constant V1_INVALIDATE_BLOCK_LOG_TOPIC =
         keccak256("BlockInvalidated(bytes32)");
+
+    bool public constant TAIKO_WHITELIST_PROVERS = false;
```

Comment:
@davidtaikocha please replace the value to "true" when you deploy the next testnet to enable prover whitelisting.

Reply:
In the future, all constants in this file start with "K_".